### ISSUE TITLE
Installing 3 packages: doRNG, pkgmaker, registry Error in if (type == "binary") { : argument is of length zero

### ISSUE THREAD

**Issue by mstaniak** [psobczyk/varclust#8]
I'm getting this error after running

```
devtools::install_github("psobczyk/varclust")
```

**Comment by sjwilczynski** [comment]
What version of R are you using?

**Comment by mstaniak** [comment]
```
- Session info -------------------------------------------------------------------------------------------
 setting  value                       
 version  R version 3.5.1 (2018-07-02)
 os       Windows >= 8 x64            
 system   x86_64, mingw32             
 ui       RStudio                     
 language (EN)                        
 collate  Polish_Poland.1250          
 ctype    Polish_Poland.1250          
 tz       Europe/Warsaw               
 date     2019-05-31                  

- Packages -----------------------------------------------------------------------------------------------
 package     * version    date       lib source                        
 assertthat    0.2.1      2019-03-21 [1] CRAN (R 3.5.3)                
 backports     1.1.4      2019-04-10 [1] CRAN (R 3.5.3)                
 base64enc     0.1-3      2015-07-28 [1] CRAN (R 3.5.0)                
 callr         3.0.0      2018-08-24 [1] CRAN (R 3.5.1)                
 cli           1.1.0      2019-03-19 [1] CRAN (R 3.5.3)                
 crayon        1.3.4      2017-09-16 [1] CRAN (R 3.5.1)                
 curl          3.2        2018-03-28 [1] CRAN (R 3.5.1)                
 desc          1.2.0      2018-05-01 [1] CRAN (R 3.5.1)                
 devtools      2.0.1      2018-10-26 [1] CRAN (R 3.5.3)                
 digest        0.6.19     2019-05-20 [1] CRAN (R 3.5.3)                
 fs            1.2.6      2018-08-23 [1] CRAN (R 3.5.1)                
 glue          1.3.1      2019-03-12 [1] CRAN (R 3.5.3)                
 magrittr      1.5        2014-11-22 [1] CRAN (R 3.5.1)                
 memoise       1.1.0      2017-04-21 [1] CRAN (R 3.5.1)                
 pkgbuild      1.0.3      2019-03-20 [1] CRAN (R 3.5.3)                
 pkgload       1.0.1      2018-10-11 [1] CRAN (R 3.5.1)                
 prettyunits   1.0.2      2015-07-13 [1] CRAN (R 3.5.1)                
 processx      3.2.0      2018-08-16 [1] CRAN (R 3.5.1)                
 ps            1.1.0      2018-08-10 [1] CRAN (R 3.5.1)                
 R6            2.4.0      2019-02-14 [1] CRAN (R 3.5.3)                
 Rcpp          1.0.1      2019-03-17 [1] CRAN (R 3.5.3)                
 remotes       2.0.2.9000 2019-04-08 [1] Github (r-lib/remotes@c462fb7)
 rlang         0.3.4      2019-04-07 [1] CRAN (R 3.5.3)                
 rprojroot     1.3-2      2018-01-03 [1] CRAN (R 3.5.1)                
 rstudioapi    0.8        2018-10-02 [1] CRAN (R 3.5.1)                
 sessioninfo   1.1.1      2018-11-05 [1] CRAN (R 3.5.3)                
 testthat      2.0.1      2018-10-13 [1] CRAN (R 3.5.1)                
 usethis       1.4.0      2018-08-14 [1] CRAN (R 3.5.2)                
 withr         2.1.2      2018-03-15 [1] CRAN (R 3.5.1)                
 yaml          2.2.0      2018-07-25 [1] CRAN (R 3.5.1)                

[1] C:/Users/mstaniak/Documents/R/win-library/3.5
[2] C:/Program Files/R/R-3.5.1/library
```

Thanks for the link, i'll look into this and check back with you

**Comment by sjwilczynski** [comment]
According to these threads ([1](https://github.com/r-lib/remotes/issues/291) and [2](https://github.com/r-lib/devtools/issues/2020)) the problem is fixed in remotes package 2.0.3 and you have version 2.0.2.9000. I hope this is the case.

**Comment by mstaniak** [comment]
thanks, it worked now
